### PR TITLE
Feature/holders component

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Refactor holders section to use the same component in different places.
Also add pagination to the holders' table.

#### What problem is this solving?
Code duplicate.

#### Screenshots or example usage
screen shot 2018-05-18 at 15 35 51

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Holders.js [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+  }
+
+  componentDidMount() {
+    this.props.getHolders({
+      checkingAccountId: this.state.checkingAccountId,
+      env: this.state.env,
+    })
+  }
+
+  componentWillReceiveProps(nextProps) {
```
Comment (leticiawanderley) [inline react/components/Holders.js]: `componentWillReceiveProps` is no longer safe, consider using `componentDidUpdate`

Comment (klzns) [reply react/components/Holders.js]: É seguro se for so pra setar o state a partir das props.

Comment (kevinch) [reply react/components/Holders.js]: Faz isso mesmo.

File: react/my-credits.js [hunk 2]
```diff
@@ -123,13 +105,7 @@ class MyCredits extends Component {
...
           checkingAccountId: createCheckingAccountId(nextProps.checkingAccount),
         },
-        () => {
-          this.getInvoices()
-          this.props.getHolders({
-            checkingAccountId: this.state.checkingAccountId,
-            env: this.state.env,
-          })
-        },
+        () => this.getInvoices(),
```
[the remark this answers is not in the document]
Comment (kevinch) [reply react/my-credits.js]: Não funciona.

**Pull request merged**